docs/howto/connect_service_with_elastic_stack.md

```diff
@@ -100,6 +100,10 @@ access.log  error.log  httpd.pid
 When you need to specify the service address, keep in mind that the Elastic stack and the service operate in Docker networks,
 so they don't communicate over the `localhost`, but use dedicated domains instead, e.g.: `elastic-package-service_apache_1:80`.
 
+6. Define the service's log path in the Kibana Fleet UI.
```

Comment:
I would mention here that this step is "optional" as it affects only logs based data streams. If you don't plan to test any logs based data streams, you don't need to do it.

Reply:
I think it is obvious that it has to do with logs related datastreams. Same goes for point 5, it would optional for someone that only wants to test logs collection.